app/views/static.py

```diff
@@ -1,25 +1,33 @@
-from flask import Blueprint
+from flask import Blueprint, g, current_app
+from flask_login import current_user
 from flask_themes2 import render_theme_template
 
-from app.helpers import template_helper
-
+from app.globals import get_metadata
+from app.utilities.schema import load_schema_from_metadata
 
 from structlog import get_logger
 
 logger = get_logger()
 
-contact_blueprint = Blueprint('contact', __name__,)
+contact_blueprint = Blueprint(name='contact', import_name=__name__)
 
 
 @contact_blueprint.route('/contact-us', methods=['GET'])
 def contact():
-    return render_theme_template('default', 'static/contact-us.html')
+    metadata = get_metadata(current_user)
```

Comment:
We should be getting this information from the session, rather than metadata. The metadata is removed on submission, the session will still hold the eq_id, form_type and ru_ref.